token-metadata/program/tests/mint_new_edition_from_master_edition_via_token.rs

```diff
@@ -307,10 +305,39 @@ mod mint_new_edition_from_master_edition_via_token {
         assert_custom_error!(result, MetadataError::EditionOverrideCannotBeZero);
     }
 
+    #[tokio::test]
+    async fn fail_to_mint_edition_num_zero() {
```

Comment:
Looking at commit history and this PR, you did not change any logic for detecting `MetadataError::EditionOverrideCannotBeZero` right?  Just adding a test case here?

Reply:
Correct.